sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj

```diff
@@ -32,6 +32,8 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+    <PackageReference Include="Microsoft.RL" VersionOverride="1.0.18120003-INTERNAL" />
+    <PackageReference Include="Newtonsoft.Json" VersionOverride="10.0.3" />
```

Comment:
We are already using System.text.Json in the sdk

Reply:
Changed.